HerPortal.BusinessLogic/ExternalServices/S3FileReader/S3FileReader.cs

```diff
@@ -29,7 +32,21 @@ ILogger<S3FileReader> logger
 
         try
         {
-            s3Client = new AmazonS3Client(RegionEndpoint.GetBySystemName(config.Region));
+            if (environment.IsEnvironment("Development"))
+            {
+                // For local development connect to a local instance of Minio
+                var clientConfig = new AmazonS3Config
+                {
+                    AuthenticationRegion = config.Region,
+                    ServiceURL = config.LocalDevOnly_ServiceUrl,
+                    ForcePathStyle = true,
+                };
+                s3Client = new AmazonS3Client(config.LocalDevOnly_AccessKey, config.LocalDevOnly_SecretKey, clientConfig);
+            }
+            else
+            {
+                s3Client = new AmazonS3Client(RegionEndpoint.GetBySystemName(config.Region));
+            }
```

Comment:
I'm not sure this class should know the details of our dev/prod setup - it might be better to set up an appropriate S3 Client using DI?

Reply:
I'll take a look at that